src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -275,6 +255,58 @@ private function dumpObjects()
         }
     }
 
+    private function fixItems()
+    {
+        $itemtype = $this->input->getOption('itemtype');
+        $item_ids = $this->input->getOption('id');
+
+        foreach ($item_ids as $item_id) {
+            $item = new $itemtype();
+            if (!$item->getFromDB($item_id)) {
+                $this->failed_items[] = $item_id;
+                continue;
+            }
+            $this->fixOneItem($item);
+        }
+    }
+
+    private function fixOneItem(CommonDBTM $item)
+    {
+        global $DB;
+
+        $itemtype = $this->input->getOption('itemtype');
+        $fields = $this->input->getOption('field');
+
+        // update the item
+        $update = [];
+        foreach ($fields as $field) {
+            $update[$field] = $this->fixOneField($item, $field);
+            $update[$field] = $DB->escape($update[$field]);
+        }
+
+        $success = $DB->update(
+            $itemtype::getTable(),
+            $update,
+            ['id' => $item->fields['id']],
+        );
+        if (!$success) {
+            $failed_items[] = $item->getID();
+        }
+    }
+
+    private function fixOneField(CommonDBTM $item, string $field)
+    {
+        $new_value = $item->fields[$field];
+
+        $new_value = $this->doubleEncoding($new_value);
+
+        if ($item::getType() == ITILFollowup::getType() && $field == 'content') {
+            $new_value = $this->emailsInOutlookGenetaredContent($new_value);
```

Comment:
This problem should exists only in **ITILFollowups**, field **content** 

The "routing" here should reduce risk of damaging data.

Reply:
(method renamed from a suggestion)